src/index.html

```diff
@@ -8,6 +8,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
stars--0 is not needed

Reply:
maybe it is needed for the tests
![Screenshot 2022-03-21 at 12 03 41](https://user-images.githubusercontent.com/79282749/159239965-f98fa805-1fa8-46dc-a38e-ce5ba448ede0.jpg)